db/migrate/20200907140351_rename_type_column_in_issues_table.rb

```diff
@@ -0,0 +1,5 @@
+class RenameTypeColumnInIssuesTable < ActiveRecord::Migration[6.0]
+  def change
+    rename_column :issues, :type, :issue_type
```

Comment:
rename the column in the previous migration by deleting from schema_migration / rollback. Only add migration of rename of change column if the db already merged. Discuss with me or your mentor.

Reply:
Doing it now

Reply:
Done